explorer/src/explorer/Main.hs

```diff
@@ -65,7 +65,7 @@ action args@Args {..} = do
     t <- currentTime
     logInfo $ sformat ("Current time is " % shown) (Timestamp t)
     nodeParams <- getNodeParams args systemStart
-    putText $ "Running using " <> show (CLI.sscAlgo commonArgs)
+    putText $ "Running using " <> show sscAlgo
```

Comment:
It seems explorer supports only GodTossing. The only usage of `sscAlgo` in this file is this line.